llvm/test/CodeGen/X86/xray-leaf-stack-alignment.ll

```diff
@@ -0,0 +1,18 @@
+; RUN: llc -mtriple=x86_64 < %s | FileCheck %s
```

Comment:
A file-level comment `;; ...` can be useful

Reply:
The test can be added to xray-custom-log.ll

Reply:
Move the test to xray-custom-log.ll ? :)

Reply:
Per discussions on https://discourse.llvm.org/t/rfc-github-pr-resolve-conversation-button/73178 , it's recommended for reviewers to resolve comments, even if they are really straightforward:)

Reply:
Ah right, moved, thanks (and thanks for the heads up re resolving comments - still getting used to using Github :-))